cmd/pow/cmd/prepare/prepare.go

```diff
@@ -36,6 +41,7 @@ func init() {
 	prepare.Flags().String("tmpdir", os.TempDir(), "path of folder where a temporal blockstore is created for processing data")
 	prepare.Flags().String("ipfs-api", "", "IPFS HTTP API multiaddress that stores the cid (only for Cid processing instead of file/folder path)")
 	prepare.Flags().Bool("json", false, "avoid pretty output and use json formatting")
+	prepare.Flags().Bool("aggregate", false, "aggregates a folder of files")
```

Comment:
New flag that indicates the files in the provided folder should be aggregated with the aggregator library.

Reply:
Thank you ❤️🙏